app/submitter/encrypter.py

```diff
@@ -20,10 +19,7 @@ class Encrypter(JWEEncrypter):
 
     # pylint: disable=maybe-no-member
     # password and key variables are dynamically assigned
-    def __init__(self,
-                 private_key=settings.EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY,
-                 private_key_password=settings.EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY_PASSWORD,
-                 public_key=settings.EQ_SUBMISSION_SDX_PUBLIC_KEY):
+    def __init__(self, private_key, private_key_password, public_key):
```

Comment:
We should check that these parameters are provided in a similar way to the `Encoder` initialisation.